lib/guardian.rb

```diff
@@ -370,7 +371,7 @@ def can_invite_to?(object, groups = nil)
       end
     end
 
-    user.has_trust_level?(TrustLevel[2])
+    user.has_trust_level?(SiteSetting.min_trust_to_allow_pm_invite.to_i)
```

Comment:
This doesn't look right to me since an `object` can either be an instance of `Topic` or something else. Also, `topic` can either be a public topic or a PM.

Reply:
TrustLevel[2]
=> 2

looks right to me @tgxworld